### ISSUE TITLE
about cvc-14 training dataset

### ISSUE THREAD

**Issue by zugofn** [sejong-rcv/MLPD-Multi-Label-Pedestrian-Detection#18]
Hi! Thanks for your great job!
I want to reproduce the performance of cvc data based on your code.
I found the data related to the cvc dataset in the [issue](https://github.com/sejong-rcv/MLPD-Multi-Label-Pedestrian-Detection/issues/12), there are only annotations for testing(such as 'CVC-14_test_all_Full.json') but no annotations for training(CVC-14_train_all_Full.json).

Can you provide relevant json file?
Have a nice day！🙂

**Comment by socome** [comment]
Thank you for your interest in our work. 

When you train the model on the CVC-14 datset, 'CVC-14_train_all_Full.json' is not needed.  In other words, you need to make the [dataloader](https://github.com/sejong-rcv/MLPD-Multi-Label-Pedestrian-Detection/blob/8e583929a97b9f959634efa69174435ad0252a57/src/datasets.py#L16) for CVC-14 dataset that can use any other type annotations file(xml, json, txt, etc.), and you can get CVC-14 annotations for training through [this website](http://adas.cvc.uab.es/elektra/enigma-portfolio/cvc-14-visible-fir-day-night-pedestrian-sequence-dataset/).  As already mentioned, we don't have the config  file and dataloader for training the CVC-14 datasets because we only organize our code for training the KAIST datasets.

**Comment by zugofn** [comment]
> Thank you for your interest in our work.
> 
> When you train the model on the CVC-14 datset, 'CVC-14_train_all_Full.json' is not needed. In other words, you need to make the [dataloader](https://github.com/sejong-rcv/MLPD-Multi-Label-Pedestrian-Detection/blob/8e583929a97b9f959634efa69174435ad0252a57/src/datasets.py#L16) for CVC-14 dataset that can use any other type annotations file(xml, json, txt, etc.), and you can get CVC-14 annotations for training through [this website](http://adas.cvc.uab.es/elektra/enigma-portfolio/cvc-14-visible-fir-day-night-pedestrian-sequence-dataset/). As already mentioned, we don't have the config file and dataloader for training the CVC-14 datasets because we only organize our code for training the KAIST datasets.

Thank you very much for your reply!!!
I would like to ask two more questions:

1.Like [dataset.py in the issue](https://github.com/sejong-rcv/MLPD-Multi-Label-Pedestrian-Detection/issues/11), do I just need to use visible annotations to train the network?
2.When I test, which one of 'CVC-14_test_all_Full.json' and 'CVC-14_test_all_visible.json' should I use? What does 'Full' mean?

Looking forward to your reply,thanks again!!

**Comment by zugofn** [comment]
@socome

**Comment by socome** [comment]
1. Yes. If you want to get more details, plz refer to paepr. 
2. You need to use  'CVC-14_test_all_visible.json' because we also follow the protocol introduced in [21] as other studies also adopt this.(refer to paper).  'CVC-14_test_all_Full.json' means the combined annotations of RGB and Thermal bbox.

**Comment by zugofn** [comment]
> 

Thank you for your reply！
Happy weekend~😊

**Comment by zugofn** [comment]
> 1. Yes. If you want to get more details, plz refer to paepr.
> 2. You need to use  'CVC-14_test_all_visible.json' because we also follow the protocol introduced in [21] as other studies also adopt this.(refer to paper).  'CVC-14_test_all_Full.json' means the combined annotations of RGB and Thermal bbox.

Thank you for your reply！
Happy weekend~😊
